.travis.yml

```diff
@@ -8,12 +8,15 @@ addons:
     sources:
     - google-chrome
     packages:
-    - google-chrome-stable
+    - google-chrome-stable xvfb
```

Comment:
What's the purpose of using "xvfb" here ?
Does it install a special version of chrome ?